File: programmers/Level1/로또의 최고 순위와 최저 순위/로또.py
def solution(lottos, win_nums):
    answer = [0, 0]
    win_count = 0
    zero_count = 0
    for i in lottos:
        if i == 0:
            zero_count += 1
        for j in win_nums:
            if i == j:
                win_count += 1
    if zero_count + win_count == 0:
        answer[0] = 6
    else:
        answer[0] = 7 - (win_count + zero_count)
        
    if win_count == 0 or win_count == 1:
        answer[1] = 6
    else:
        answer[1] = 7 - win_count
    return answer

File: programmers/Level1/로또의 최고 순위와 최저 순위/로또.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int *solution(int *lottos, int lottos_len, int *win_nums, int win_nums_len)
{
    int* answer = (int*)malloc(2 * sizeof(int));
    int zero_count = 0;
    int i = -1;
    int win_count = 0;
    
    while (++i < lottos_len)
    {
        if (lottos[i] == 0)
            zero_count++;
        int j = -1;
        while (++j < win_nums_len)
            if (lottos[i] == win_nums[j])
                win_count++;
    }
    if (zero_count + win_count == 0)
        answer[0] = 6;
    else
        answer[0] = 7 - (zero_count + win_count);
    if (win_count == 0)
        answer[1] = 6;
    else
        answer[1] = 7 - win_count;
    return answer;
}
